include/deal.II/base/cuda.h

```diff
@@ -52,17 +53,25 @@ namespace Utilities
       Handle(Handle const &) = delete;
 
       /**
-       * Destructor. Destroy the handles and free all the memory allocated by
-       * GrowingVectorMemory.
+       * Destructor. Destroy the handles.
        */
       ~Handle();
 
+      /**
+       * Pointer to an opaque cuSolverDN context.
+       * The handle must be passed to every cuSolverDN library function.
+       */
       cusolverDnHandle_t cusolver_dn_handle;
 
+      /**
+       * Pointer to an opaque cuSolverSP context.
+       * The handle must be passed to every cuSolverSP library function.
+       */
       cusolverSpHandle_t cusolver_sp_handle;
 
       /**
-       * Handle to the cuSPARSE library.
+       * Pointer to an opaque cuSparse context.
```

Comment:
The name of the library is `cuSPARSE` (see [here](https://developer.nvidia.com/cusparse)) not `cuSparse`.

Reply:
Fixed.